docs/validators/node-configuration.md

```diff
@@ -0,0 +1,84 @@
+---
+id: node-configuration
+title: Node Configuration
+---
+
+## Introduction
+
+The node relies on some configuration files that are meant to allow the node operator to easily change some values 
+that won't require a code change, a new release or so on. 
+
+## Configuration files
+
+All the configuration files are by default placed into the `config` directory that resides near the node's binary. The paths can be changed 
+by using the node's CLI flags.
+
+:::important
+Note all the configuration values can be user-defined. For example, it is perfectly fine if a node operator increases the size of a cacher or sets an Elasticsearch instance, but changing the genesis total supply, for example, will lead to an inconsistent state as compared to the Network.
+:::
+
+Below you can find an example on how the configuration files should be present (at the time of writing the documentation).
```

Comment:
Perhaps we can specify the current node version rather than "at the time of writing the docs".

Reply:
good suggestion. applied